### ISSUE TITLE
2.0.3 build issue

### ISSUE THREAD

**Issue by chenrui333** [ttscoff/mdless#91]
👋 trying to build the latest release, but run into some build issue. The error log is as below:

```
  ERROR:  Could not find a valid gem 'mdless-2.0.3.gem' (>= 0) in any repository
```

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/6963602943/job/18949890650
relates to Homebrew/homebrew-core#155208

**Comment by ttscoff** [comment]
Fixed the discrepancy between develop and master and released 2.0.4. Let 
me know if that fixes it for you.

-Brett

**Comment by chenrui333** [comment]
@ttscoff thanks for the quick turnaround, looks like it works for us, https://github.com/Homebrew/homebrew-core/pull/155235
